e2e/tslint.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
This was imported en masse from the same file in automate-ui (so we use a consistent rule set), less the extensions that are only needed in an Angular project.